test.js

```diff
@@ -20,7 +20,7 @@ if (!process.env.CI) {
 
 			vol.get(function (err, level) {
 				t.assert(!err, err);
-				t.assert(level === 0.3);
+				t.assert(0.25 < level && level < 0.35);
```

Comment:
This wouldn't work
The issue is because the volume is not set properly (it's set 1% higher)
I could remove 1% when I set the volume but I don't think it's a great idea (as it could come from my configuration)